docs/configuration.md

```diff
@@ -0,0 +1,316 @@
+    kind: SelfNodeRemediationTemplate
+    namespace: <SNR namespace>
+    name: self-node-remediation-resource-deletion-template
+  escalatingRemediations:
+    - remediationTemplate:
+        apiVersion: superfast.medik8s.io/v1alpha1
+        name: superfast
+        namespace: openshift-operators
+        kind: SuperFastRemediationTemplate
+      order: 1
+      timeout: 60s
+    - remediationTemplate:
+        apiVersion: normal.medik8s.io/v1alpha1
+        name: normal
+        namespace: openshift-operators
+        kind: NormalRemediationTemplate
+      order: 2
+      timeout: 300s
+  minHealthy: "51%"
+  unhealthyConditions:
+    - type: Ready
+      status: "False"
+      duration: 300s
+    - type: Ready
+      status: Unknown
+      duration: 300s
+```
+
+### Spec Details
+
+| Field                    | Mandatory                             | Default Value                                                                                   | Description                                                                                                                                                                                    |
+|--------------------------|---------------------------------------|-------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| _selector_               | yes                                   | n/a                                                                                             | A [LabelSelector](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements) for selecting nodes to observe. See details below.  | 
+| _remediationTemplate_    | yes but mutually exclusive with below | n/a                                                                                             | A [ObjectReference](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/object-reference/) to a remediation template provided by a remediation provider. See details below. |
+| _escalatingRemediations_ | yes but mutually exclusive with above | n/a                                                                                             | A list of ObjectReferences to a remediation template with order and timeout. See details below.                                                                                                |
+| _minHealthy_             | no                                    | 51%                                                                                             | The minimum number of healthy nodes selected by this CR for allowing further remediation. Percentage or absolute number.                                                                       |
```

Comment:
Hmm I though `minHealthy` is mandatory.
Do we have business logic for the use case where it does not exist ? (for example if default value is deleted)

Reply:
If there is no possible use case where this value is empty doesn't it mean it's mandatory ?
(That's at least how I understand mandatory as opposed to be needed to be set by the user)